### ISSUE TITLE
Custom partitioning requires to use the change button

### ISSUE THREAD

**Issue by seb128** [canonical/ubuntu-desktop-installer#1203]
The old desktop installer let you right or double click on a partition line to edit it which requires less mouse movement than selecting and having to go to the button

**Comment by jpnurmi** [comment]
@seb128 @ken-vandine @elioqoshi 

Due to how gesture negotiation works in Flutter, handling double-clicks in the manual partitioning table adds 300ms delay to single-clicks. There's a well-noticeable and somewhat annoying delay when selecting rows or checking the "Format" checkboxes. Adding new partitions and editing existing ones by double-clicking is much more convenient, but it makes all other interactions within the table feel sluggish.

Should we give it a try so you can all get a feeling? We can revert it the next day if you don't like it...

![image](https://user-images.githubusercontent.com/140617/211852579-85724967-99bc-407c-9b23-db63d0e5a2be.png)

**Comment by elioqoshi** [comment]
Can we have both options? Also double clicking and selecting + button?

**Comment by jpnurmi** [comment]
Yes, the old select+button functionality will be kept. :+1: Double-click is just a shortcut (with an unfortunate side-effect).
